mongodbatlas/resource_mongodbatlas_network_peering.go

```diff
@@ -61,9 +61,8 @@ func resourceMongoDBAtlasNetworkPeering() *schema.Resource {
 
 			"provider_name": {
 				Type:         schema.TypeString,
-				Optional:     true,
+				Required:     true,
```

Comment:
I don't think this is being used when sending the update request. Line 426 doesn't include provider_name

Reply:
It should be within peer, also see line 389, is this correct @marinsalinas ?

Reply:
If I'm reading line 389 correctly, providerName will only be included if it changed but it should always be included.

Reply:
> I don't think this is being used when sending the update request. Line 426 doesn't include provider_name

That's right, let me add it to the request.

Reply:
I've added it on the last commit